odoo/cli/server.py

```diff
@@ -136,6 +138,11 @@ def main(args):
         for db_name in preload:
             try:
                 odoo.service.db._create_empty_database(db_name)
+            except ProgrammingError as err:
+                if err.pgcode == errorcodes.INSUFFICIENT_PRIVILEGE:
+                    _logger.warning("Could not determine if database exists: %s" % err)
```

Comment:
@Yenthe666 I realize that ;-) The only reason we were considering INFO is to prevent all builds failing because of this warning, on build environment with restricted database access (which may become the case on runbot in the future)